### ISSUE TITLE
Separate issues from mailing list issues

### ISSUE THREAD

**Issue by StefanCodes** [angular-ui-tree/angular-ui-tree#528]
The issue tracker has a lot of how-to questions and other issues that really aren't Issues - they're inquiries that don't have a better place.

I propose we/someone:
* Setup a user mailing list/group for general inquiries that aren't development tasks
* Flesh out the wiki with more examples and an FAQ
* Moderate issues and close any that aren't development tasks (with a link to the documentation where they can find the answer)

I'm happy to contribute to a wiki/FAQ and create some examples (when I've got some free time). I think a mailing list/forum should be setup and owned by a project maintainer or owner, though.

What do other people think?

**Comment by Voles** [comment]
Hi @StefanCodes, thanks for your feedback!
You're right that a lot of issues are not always related to the development of the project.

What do you think of setting using [Slack](https://slack.com/) to create some sort of community to help with Q&A? I've seen similar groups for Angular and EmberJS communities. Or do you prefer a mailing list?

I have simplified the steps to update the website by adding a Gulp task (`$ gulp deploy`), see #532.
We can add a few more examples with common scenario's of our users.

The README can also be optimised by moving some parts of it to a separate page on the website.

What do you think?

**Comment by Voles** [comment]
BTW I can give you push rights if you're interested in contributing to the project :-)

**Comment by StefanCodes** [comment]
I've used Slack a little bit in the past and was quite happy with how it worked. Mailing lists are fine, too - I don't have much of an opinion of what tools are in use beyond people being able to find and use them.

When I've got a free afternoon I'll pore through the existing issues list and flesh out a Common Questions page for the wiki (have we seen enough drag and drop questions yet? :P). I do still think a separate forum/list for users is a good idea. It's a shame GitHub doesn't have anything like that.

Thanks for push rights! :)
